boards/shields/rtkmipilcdb00000be/doc/index.rst

```diff
@@ -0,0 +1,61 @@
+.. _rtkmipilcdb00000be:
+
+RTKMIPILCDB00000BE MIPI Display
+############################
```

Comment:
Underlines need to be same width as the title. This is likely why your documentation build is failing.